functions/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "nwhacks2019",
+  "version": "0.1.0",
+  "description": "The backend (cloud functions) for nwHacks 2019",
+  "author": "nwHacks2019",
+  "license": "MIT",
+  "scripts": {
+    "pick:dev": "firebase login && firebase use development",
+    "pick:prod": "firebase login && firebase use production",
+    "lint": "eslint .",
+    "serve": "yarn pick:dev && firebase serve --only functions",
+    "shell": "yarn pick:dev && firebase functions:shell",
+    "start": "yarn pick:dev && yarn run shell",
+    "deploy:dev": "yarn pick:dev && firebase deploy --only functions",
+    "deploy:prod": "yarn pick:prod && firebase deploy --only functions",
+    "logs": "firebase functions:log"
```

Comment:
travis is looking for a `test` script here, can probably copy the one that used to be in `serverless` or add a simple one that exits with `0` to appease travis for now